qubes/tests/__init__.py

```diff
@@ -389,10 +389,17 @@ class _clear_ex_info(contextlib.ContextDecorator):
     """Remove local variables reference from tracebacks to allow garbage
     collector to clean all Qubes*() objects, otherwise file descriptors
     held by them will leak"""
+    def __init__(self, result_callback=None):
+        self._result_callback = result_callback
+
     def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
+        if self._result_callback:
+            self._result_callback(bool(
+                not exc_type or isinstance(exc_type, unittest.SkipTest)
+            ))
```

Comment:
Why the call to `bool` here?  It looks redundant.

Reply:
Earlier version needed it, now can be removed.